x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -157,6 +225,123 @@ export function XyToolbar(props: VisualizationToolbarProps<State>) {
+              isFullWidth
+              type="multi"
+            />
+          </EuiFormRow>
+          <EuiHorizontalRule margin="s" />
+          <EuiTitle size="xxs">
+            <span>
+              {i18n.translate('xpack.lens.xyChart.axisTitles', { defaultMessage: 'Axis titles' })}
+            </span>
+          </EuiTitle>
+          <EuiFormRow
+            display="columnCompressed"
+            label={
+              <EuiFlexGroup gutterSize="s">
+                <EuiFlexItem>X-axis</EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiSwitch
+                    data-test-subj="lnsshowXAxisTitleSwitch"
+                    showLabel={false}
+                    label={i18n.translate('xpack.lens.xyChart.showXAxisTitleLabel', {
+                      defaultMessage: 'show X-axis Title',
+                    })}
+                    onChange={({ target }) =>
+                      props.setState({ ...props.state, showXAxisTitle: target.checked })
+                    }
+                    checked={props.state?.showXAxisTitle ?? true}
+                  />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            }
+          >
+            <EuiFieldText
+              data-test-subj="lnsXAxisTitle"
+              compressed
+              placeholder={i18n.translate('xpack.lens.xyChart.overwriteXaxis', {
+                defaultMessage: 'Overwrite X-axis title',
+              })}
+              value={xtitle}
+              disabled={
+                props.state && 'showXAxisTitle' in props.state ? !props.state.showXAxisTitle : false
+              }
+              onChange={({ target }) => onXTitleChange(target.value)}
+              aria-label={i18n.translate('xpack.lens.xyChart.overwriteXaxis', {
+                defaultMessage: 'Overwrite X-axis title',
+              })}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            display="columnCompressed"
+            label={
+              <EuiFlexGroup gutterSize="s">
+                <EuiFlexItem>Y-axis</EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiSwitch
+                    data-test-subj="lnsShowYAxisTitleSwitch"
+                    showLabel={false}
+                    label={i18n.translate('xpack.lens.xyChart.ShowYAxisTitleLabel', {
+                      defaultMessage: 'Show Y-axis Title',
+                    })}
+                    onChange={({ target }) =>
+                      props.setState({ ...props.state, showYAxisTitle: target.checked })
+                    }
+                    checked={props.state?.showYAxisTitle ?? true}
+                  />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            }
+          >
+            <EuiFieldText
+              data-test-subj="lnsYAxisTitle"
+              compressed
+              placeholder={i18n.translate('xpack.lens.xyChart.overwriteYaxis', {
+                defaultMessage: 'Overwrite Y-axis title',
```

Comment:
Instead of this placeholder message, can the value be the actual y-axis title that is currently showing before the user changes it? Same for the x-axis title

Reply:
After discussing it with Caroline we agreed that the flow for the titles will be as follows:
1. User opens the popover and sees in the input the default value which is the label
2. User changes the title -> the chart title also changes
3. The user empties the input field: the chart title changes to the default and a placeholder appears to the text. User closes the popover.
4. If the user opens again the popover, she will see the label in the input text field

Working on this right now. Will have an updated PR soon.

Reply:
@cchaos the new flow now is: 
![Lens_new_user_flow_settings_popover](https://user-images.githubusercontent.com/17003240/88159692-52e04e00-cc16-11ea-8e19-8c5d17fef1fa.gif)